infra-as-code/bicep/machinelearning.bicep

```diff
@@ -159,28 +158,22 @@ resource aiHub 'Microsoft.MachineLearningServices/workspaces@2024-07-01-preview'
     friendlyName: 'Azure OpenAI Chat Hub'
     description: 'Hub to support the Microsoft Learn Azure OpenAI basic chat implementation. https://learn.microsoft.com/azure/architecture/ai-ml/architecture/basic-openai-e2e-chat'
     publicNetworkAccess: 'Enabled' // Production readiness change: The "Baseline" architecture adds ingress and egress network control over this "Basic" implementation.
-    ipAllowlist: []
-    serverlessComputeSettings: null
-    enableServiceSideCMKEncryption: false
     managedNetwork: {
-      isolationMode: 'Disabled' // Production readiness change: The "Baseline" architecture adds ingress and egress network control over this "Basic" implementation.
+      isolationMode: 'Disabled'
     }
-    allowRoleAssignmentOnRG: false // Require role assignments at the resource level.
     v1LegacyMode: false
     workspaceHubConfig: {
-      defaultWorkspaceResourceGroup: resourceGroup().id  // Setting this to the same resource group as the workspace
+      defaultWorkspaceResourceGroup: resourceGroup().id // Setting this to the same resource group as the workspace
     }
 
     // Default settings for projects
     storageAccount: aiStudioStorageAccount.id
     containerRegistry: containerRegistry.id
-    systemDatastoresAuthMode: 'identity'
-    enableSoftwareBillOfMaterials: true

```

Comment:
Can you say more about removing these? Did they get removed from the API?

Reply:
In `2024-10-01` yes but it seems it was added back in `2025-01-01-preview`.

Reply:
Let's roll forward to 2025-01-01-preview -- unless you know of a reason not to?